pom.xml

```diff
@@ -133,6 +133,12 @@
             <version>2.2</version>
         </dependency>
 
+		<dependency>
+			<groupId>org.owasp.encoder</groupId>
+			<artifactId>encoder</artifactId>
```

Comment:
Isn't this already in the openmrs platform? https://github.com/openmrs/openmrs-core/blob/2.2.0/web/pom.xml#L116-L120

Reply:
So it is! I just updated to use the version from WebUtils in core.